llvm/test/Transforms/LoopVectorize/vector-to-scalar-cast.ll

```diff
@@ -0,0 +1,41 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt --force-widen-divrem-via-safe-divisor=false -passes=loop-vectorize -S --debug-only=loop-vectorize < %s 2>&1 | FileCheck %s
+
```

Comment:
Might be good to add a comment like: "The test was crashing earlier due to a vectorization attempt; vector-to-scalar cast is now forbidden in the legalizer, and the test isn't a vectorization candidate now".

Reply:
Good idea, added, thanks!